### PR TITLE
attempt to sync help docs

### PR DESCRIPTION
**Pull request by nhayfield**
@ZPain8464 Tried to fix the merge issues.
Not sure about Databroker settings

### CODE REVIEW

File: content/docs/reference/reference.json [hunk 1]
```diff
@@ -620,12 +724,6 @@
     "services": ["proxy"],
     "type": "bool"
   },
-  "databroker-settings": {
```
Comment (ZPain8464) [inline content/docs/reference/reference.json:623]: Definitely need this one to stay.

File: content/docs/reference/reference.json [hunk 2]
```diff
@@ -135,22 +135,101 @@
...
     "id": "identity-provider-settings",
     "title": "Identity Provider Settings",
     "path": "/identity-provider-settings",
     "description": "Configure and self-host your own Identity Provider with Pomerium's Identity Provider settings.",
     "type": "string"
   },
+  "cookie-secure": {
+    "id": "cookie-secure",
+    "title": "Secure Cookie",
+    "path": "/cookie-secure",
```
Comment (ZPain8464) [inline content/docs/reference/reference.json:148]:
```suggestion
    "path": "/cookies#cookie-secure",
```

File: content/docs/reference/reference.json [hunk 3]
```diff
@@ -135,22 +135,101 @@
...
+    "title": "HTTPS only",
+    "path": "/cookies#cookie-http-only",
+    "description": "If true, instructs browsers to only send user session cookies over HTTPS.",
+    "services": [],
+    "type": "bool"
+  },
+  "identity-provider": {
+    "id": "identity-provider",
+    "title": "Identity Provider",
+    "path": "/../enterprise/identity-providers",
```
Comment (ZPain8464) [inline content/docs/reference/reference.json:164]:
```suggestion
    "path": "/docs/identity-providers",
```

File: content/docs/reference/reference.json [hunk 4]
```diff
@@ -135,22 +135,101 @@
...
+    "description": "Choose from a list of supported IDPs.",
+    "type": "string",
+    "enterpriseOnly": true
+  },
+  "idp-options-auth0": {
+    "id": "idp-options-auth0",
+    "path": "/../enterprise/identity-providers/auth0",
```
Comment (ZPain8464) [inline content/docs/reference/reference.json:171]:
```suggestion
    "path": "/docs/identity-providers/auth0",
```

File: content/docs/reference/reference.json [hunk 5]
```diff
@@ -135,22 +135,101 @@
...
+    "title": "Identity Provider Auth0",
+    "description": "Click for information on Auth0",
+    "type": "string",
+    "enterpriseOnly": true
+  },
+  "idp-options-azure": {
+    "id": "idp-options-azure",
+    "path": "/../enterprise/identity-providers/azure",
```
Comment (ZPain8464) [inline content/docs/reference/reference.json:179]:
```suggestion
    "path": "/docs/identity-providers/azure",
```

File: content/docs/reference/reference.json [hunk 6]
```diff
@@ -135,22 +135,101 @@
...
+    "title": "Identity Provider Azure",
+    "description": "Click for information on Azure",
+    "type": "string",
+    "enterpriseOnly": true
+  },
+  "idp-options-github": {
+    "id": "idp-options-github",
+    "path": "/../enterprise/identity-providers/github",
```
Comment (ZPain8464) [inline content/docs/reference/reference.json:187]:
```suggestion
    "path": "/docs/identity-providers/github",
```

File: content/docs/reference/reference.json [hunk 7]
```diff
@@ -135,22 +135,101 @@
...
+    "title": "Identity Provider Github",
+    "description": "Click for information on Github",
+    "type": "string",
+    "enterpriseOnly": true
+  },
+  "idp-options-gitlab": {
+    "id": "idp-options-gitlab",
+    "title": "Identity Provider Gitlab",
+    "path": "/../enterprise/identity-providers/gitlab",
```
Comment (ZPain8464) [inline content/docs/reference/reference.json:196]:
```suggestion
    "path": "/docs/identity-providers/gitlab",
```

File: content/docs/reference/reference.json [hunk 8]
```diff
@@ -135,22 +135,101 @@
...
+    "description": "Click for information on Gitlab",
+    "type": "string",
+    "enterpriseOnly": true
+  },
+  "idp-options-google": {
+    "id": "idp-options-google",
+    "path": "/../enterprise/identity-providers/google",
```
Comment (ZPain8464) [inline content/docs/reference/reference.json:203]:
```suggestion
    "path": "/docs/identity-providers/google",
```

File: content/docs/reference/reference.json [hunk 9]
```diff
@@ -135,22 +135,101 @@
...
+    "title": "Identity Provider Google",
+    "description": "Click for information on getting a service account from Google.",
+    "type": "string",
+    "enterpriseOnly": true
+  },
+  "idp-options-okta": {
+    "id": "idp-options-okta",
+    "path": "/../enterprise/identity-providers/okta",
```
Comment (ZPain8464) [inline content/docs/reference/reference.json:211]:
```suggestion
    "path": "/docs/identity-providers/okta",
```

File: content/docs/reference/reference.json [hunk 10]
```diff
@@ -135,22 +135,101 @@
...
+    "title": "Identity Provider Okta",
+    "description": "Click for information on Okta.",
+    "type": "string",
+    "enterpriseOnly": true
+  },
+  "idp-options-onelogin": {
+    "id": "idp-options-onelogin",
+    "path": "/../enterprise/identity-providers/one-login",
```
Comment (ZPain8464) [inline content/docs/reference/reference.json:219]:
```suggestion
    "path": "/docs/identity-providers/one-login",
```

File: content/docs/reference/reference.json [hunk 11]
```diff
@@ -135,22 +135,101 @@
...
+    "title": "Identity Provider Onelogin",
+    "description": "Click for information on Onelogin.",
+    "type": "string",
+    "enterpriseOnly": true
+  },
+  "idp-options-ping": {
+    "id": "idp-options-ping",
+    "path": "/../enterprise/identity-providers/ping",
```
Comment (ZPain8464) [inline content/docs/reference/reference.json:227]:
```suggestion
    "path": "/docs/identity-providers/ping",
```

File: content/docs/reference/reference.json [hunk 12]
```diff
@@ -165,6 +244,22 @@
     "description": "Minimum delay between requests to the IdP Directory.",
     "type": "string"
   },
+  "javascript-security": {
+    "id": "javascript-security",
+    "title": "Javascript Security",
+    "path": "/javascript-security",
```
Comment (ZPain8464) [inline content/docs/reference/reference.json:250]:
```suggestion
    "path": "/docs/reference/cookies#cookie-http-only",
```

File: content/docs/reference/reference.json [hunk 13]
```diff
@@ -165,6 +244,22 @@
...
+    "description": "If true, prevents javascript in browsers from reading user session cookies.",
+    "services": [],
+    "type": "bool"
+  },
+  "expiration": {
+    "id": "expiration",
+    "title": "Expiration",
+    "path": "/expiration",
```
Comment (ZPain8464) [inline content/docs/reference/reference.json:258]:
```suggestion
    "path": "/docs/reference/cookies#cookie-expiration",
```

File: content/docs/reference/reference.json [hunk 14]
```diff
@@ -891,6 +997,13 @@
     "services": [],
     "type": "bool"
   },
+  "signout-redirect-url": {
+    "id": "signout-redirect-url",
+    "title": "Signout Redirect URL",
+    "path": "/routes/signout-redirect-url",
```
Comment (ZPain8464) [inline content/docs/reference/reference.json:1003]:
```suggestion
    "path": "/signout-redirect-url",
```

File: content/docs/reference/reference.json [hunk 15]
```diff
@@ -748,6 +846,14 @@
     "short_description": "Only match requests with an exact match for this path",
     "type": "string"
   },
+  "policy-explanation": {
+    "id": "policy-explanation",
+    "title": "Explanation",
+    "description": "Adds an explanation to the user when they hit a 403 Unauthorized Endpoint that has this policy.",
+    "path": "/policy/policy-explanation",
```
Comment (ZPain8464) [inline content/docs/reference/reference.json:853]:
```suggestion
    "path": "/policy-explanation",
```

File: content/docs/reference/reference.json [hunk 16]
```diff
@@ -748,6 +846,14 @@
...
+    "services": [],
+    "type": "string"
```
Comment (ZPain8464) [inline content/docs/reference/reference.json:855]:
```suggestion
    "type": "string",
    "enterpriseOnly": "true"
```

File: content/docs/reference/reference.json [hunk 17]
```diff
@@ -917,6 +1030,13 @@
     "services": ["proxy"],
     "type": "bool"
   },
+  "tls-server-name": {
```
Comment (ZPain8464) [inline content/docs/reference/reference.json:1033]: Cut this setting.

**Review by ZPain8464 — APPROVED** [review]

File: content/docs/reference/reference.json (hunk 4 above)
Comment (nhayfield) [reply content/docs/reference/reference.json:171]: actually you don't need the /docs part with these

**Pull request merged**